### DISCUSSION TITLE
Load with custom transform

### DISCUSSION THREAD

**Discussion by Aadmaa** [Nikaple/nest-typed-config#77] [Q&A] [answered]
Is it possible to use one of the standard loaders (e.g dotenvLoader) followed by a custom function to transform the environment variables into the expected Schema shape? 

In other words, the dotenvLoader would not return the correct schema, but would load the raw environment, and the transformer would return the object matching the final schema.

**Comment by Nikaple** [comment, marked as answer]
You can `normalize` it, check out [this chapter of documentation](https://github.com/Nikaple/nest-typed-config#type-casting-on-environment-variables). Maybe I should rearrange the section title a bit?

**Comment by Aadmaa** [comment]
Perfect. Yes, it's somehow easy to miss that section, but I should have seen it. I'm not sure what to call it. Maybe "Transforming Loaded Data"?
